src/ucp/tag/rndv.c

```diff
@@ -190,6 +279,15 @@ static void ucp_rndv_complete_rndv_get(ucp_request_t *rndv_req)
         uct_rkey_release(&rndv_req->send.rndv_get.rkey_bundle);
     }
 
+    if (rndv_req->send.rndv_get.mrail) {
+        mrail = rndv_req->send.rndv_get.mrail;
+        for(i = 0; i < sizeof(mrail->rail) / sizeof(mrail->rail[0]); i++) {
```

Comment:
is `i < UCP_MAX_RAILS` the same?

Reply:
right, but usage of static array length is more accurate and more universal. unfortunately built-in ucs_static_array_size could not be compiled by ICC

Reply:
I do not see the difference if this array is defined as `rail[MAX_RAILS]` but if it will be redefined with smth like `rail = calloc(MAX_RAILS, sizeof(...))` using of the constant is more safety.

Reply:
in case if number of elements will be defined as more complex value (for example few multi-rail constants are used) current implementation will work properly, your suggestion will require make appropriate changes in all places where MAX_RAILS constant is used. Let's allow compiler make it's work :)

Reply:
Then MAX_RAIL should be defined as `#define MAX_RAILS (a+b+c)` :)